src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md

```diff
@@ -18,14 +18,6 @@ to scrape.
   Grafana](../../docs/metrics/getting-started-prometheus-grafana/README.md)
   tutorial for more information.
 
-<!-- This comment is to make sure the two notes above and below are not merged -->
-
-> [!NOTE]
```

Comment:
Couple of things to note
1. Unless we cover exemplar for monotonic SUM, lets keep some note here about Exemplars are only supported for ExplicitHistograms
2. Maybe mention that Exemplar support is only when OpenMetrics format is requested?

Reply:
I can add monotonic sum support in this PR - let me see how simple that would be.

As for exemplar support, these are exported regardless of whether OpenMetrics format is requested. Should I change this to only export when the Accept header is OpenMetrics?